src/Shell/Helper/TableHelper.php

```diff
@@ -114,7 +114,11 @@ protected function _render(array $row, array $widths, array $options = []): void
             if (!empty($options['style'])) {
                 $column = $this->_addStyle($column, $options['style']);
             }
-            $out .= '| ' . $column . str_repeat(' ', $pad) . ' ';
+            if (!is_null($column) && preg_match('#<text-right>.*</text-right>#', $column)) {
```

Comment:
```suggestion
            if ($column !== null && preg_match('#<text-right>.+</text-right>#', $column)) {
```

Reply:
What will happen when someone does `<text-right>some</text-right>text`?

Reply:
Thank you for pointing out the code.
I took in the points that you pointed out.

When I tried `<text-right>some</ text-right>text`, it was right-justified, including the text outside the tag.
```
$outputs = [
    ['Item', 'Price per piece (yen)'],
    ['Apple', '<text-right>some</text-right>text'],
    ['Orange', '100'],
];
$io->helper('Table')->output($outputs);
```
![image](https://user-images.githubusercontent.com/48991931/97931131-4f7d6680-1db0-11eb-83f3-3420fd35bdab.png)

Reply:
I assumed that would be the output. I don't think it is expected based on the markup text though. The CLI markup language borrows conventions from HTML which would generate results like `text      some` instead. We could raise an exception if there is text outside the alignment tags as it is likely an error.

It would also be good to ensure formatting like `<text-right><comment>blue</comment>normal</text-right>` behaves correctly as well.